test/semantics/resolve30.f90

```diff
@@ -28,16 +28,3 @@ subroutine s2
     y = 1
   end block
 end
-
-subroutine s3
-  integer j
-  block
-    import, only: j
-    type t
-      !ERROR: 'i' from host scoping unit is not accessible due to IMPORT
-      real :: x(10) = [(i, &
-        !ERROR: 'i' from host scoping unit is not accessible due to IMPORT
-        i=1,10)]
-    end type
-  end block
-end subroutine
```

Comment:
Was this a bad test?

Reply:
I believe so.  The scope of a statement entity is simply the statement in which it appears, and the rules about conflicts with the containing scope's use of the same name shouldn't apply here since `i` doesn't appear in the outer scope.